### ISSUE TITLE
Plugin doesn't support this project's cordova-ios version. cordova-ios: 3.8.0, failed version requirement: >=4.0.0-dev

### ISSUE THREAD

**Issue by gruppjo** [mwaylabs/generator-m#205]
When adding ios the following error is produced:

```
Couldn't finish generator: 
CordovaError: Plugin doesn't support this project's cordova-ios version. cordova-ios: 3.8.0, failed version requirement: >=4.0.0-dev
```

**Comment by gruppjo** [comment]
Apparently this is a bug in the new plugin release of `cordova-plugin-whitelist-1.1.0`
Release notes: https://cordova.apache.org/news/2015/06/22/plugins-release.html
Stackoverflow issues: 
- http://stackoverflow.com/questions/30991828/cordova-plugin-whitelist-failed-asking-for-cordova-ios-4
- http://stackoverflow.com/questions/30696234/can-not-add-ios-platform-on-cordova
Cordova issue: https://issues.apache.org/jira/browse/CB-8898

**Comment by lordgreg** [comment]
Maybe a suggestion as I've also experienced this issue. If the generator fails, it could ask user at the end, if it should clear the created files so we get the blank directory again?

**Comment by shpoont** [comment]
@gruppjo is there any difference in the generated code between project created with your workaround and project created with fixed whitelist 1.1.0?

**Comment by gruppjo** [comment]
@lordgreg, I agree that would be convenient, but I don't think we're going to implement this. This is an issue with cordova which will be fixed soon. It shouldn't happen in the first place. More info here: https://issues.apache.org/jira/browse/CB-8898. 
**I do realise this is a very cumbersome workaround right now. However** it has proven more beneficial to dedicate our time on things that actually improve the generator rather than fixing things that will be fixed soon elsewhere (see the `gulp cordova-install` command we introduced in 1.0.0 and removed in 1.1.0 because cordova implemented this 5.0.0. Please be patient on this on and be assured that the time we save on not implementing this will come in form of other developments as a reward to you.
- 
@shpoont the whitelist plugin version is different. Anything else is the same

**Comment by gruppjo** [comment]
issue is resolved. Apparently cordova rolled back the version of the `cordova-plugin-whitelist` to `1.0.0` which is compatible with `cordova-ios` `v3.8.0`! 

https://issues.apache.org/jira/browse/CB-9232
